cmd/system-probe/modules/compliance.go

```diff
@@ -0,0 +1,95 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package modules
```

Comment:
I think you might want a build tag `//go:build linux` here?

Reply:
I just noticed the increase in imports for windows system-probe

Reply:
`compliance_test.go` too

Reply:
ce919296b1343351f7c75728d5f4896a3892b0ed